test/lint.js

```diff
@@ -94,7 +91,7 @@ function load(...files) {
 }
 
 if (argv.files) {
-  load(argv.files);
+  load.apply(undefined, argv.files);
```

Comment:
For:
```js
load(...argv.files);
```
error TS2488: Type 'unknown' must have a '[Symbol.iterator]()' method that returns an iterator.